testing/nginx-naxsi/nginx.initd

```diff
@@ -1,67 +1,40 @@
 #!/sbin/openrc-run
+supervisor=supervise-daemon
 
 description="Nginx http and reverse proxy server"
 extra_started_commands="reload reopen upgrade"
 
 cfgfile=${cfgfile:-/etc/nginx/nginx.conf}
-pidfile=/run/nginx/nginx.pid
+pidfile=/run/nginx/$SVCNAME.sd.pid
 command=/usr/sbin/nginx
 command_args="-c $cfgfile"
+command_args_foreground='-g "daemon off;"'
 required_files="$cfgfile"
 
 depend() {
-	need net
```

Comment:
Okay, then please open an issue on [bugs.alpinelinux.org](https://bugs.alpinelinux.org/projects/alpine/issues). It’s not a problem of this runscript, but of our OpenRC setup for LXC. OpenRC can deal with different requirements of various platforms, I think that this can be somehow solved using keywords.